charts/mlsstats/templates/secret.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Release.Name }}
+  labels:
+    app: ldap-scim-bridge
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+type: Opaque
+stringData:
+  config.yaml: |
+{{ toYaml .Values.config | indent 4 }}
```

Comment:
Is the whole config a secret? Usually we use `.Values.secrets` and write each secret here. It is more work but makes things clearer for someone figuring out what they need to provide.
I also don't see the structure of this config specified in the `values.yaml`, so not sure what needs to be provided by the user.